### ISSUE TITLE
Software with the Prism3D engine does not start

### ISSUE THREAD

**Issue by BridgeHeadland** [joncampbell123/dosbox-x#4933]
### Describe the bug

![image](https://github.com/joncampbell123/dosbox-x/assets/85258509/3e39bf69-3e03-451c-8750-004fb75d80fc)
When I start a game, or software, with Prism3D engine, it won't start and I get this above message.
I have only tried running Duke Nukem: Manhattan Project. I don't know if it only applies to this Prism3D game, or Prism3D software in general. Could it be because the S3 ViRGE VX is not good enough for Prism3D?
I know the S3 ViRGE VX is experimental, but I still think it's good anyway.

### Steps to reproduce the behaviour

Install Windows 9x in DOSBox-X, unless you have already done so. Preferably Windows 98SE. Along with a Prism3D engine software, you install Directx from December 2006, if it's W98SE you have installed (I don't know of anything better that supports). Once everything is done, run the software with the Prism3D engine and see what happens. You must have set the CPU type to pentium_iii, core to dynamic and machine to svga_s3virgevx.

### Expected behavior

I expect software with the Prism3D engine to work properly in Windows 9x (in DOSBox-X, of course), preferably all related software for everyone's sake.

### What operating system(s) this bug have occurred on?

Windows 10 Home 2H22

### What version(s) of DOSBox-X have this bug?

 Very latest nightly builds, 64-bit MinGW portable builds (for 7+), SDL2

### Have you checked that no similar bug report(s) exist?

- [X] I have searched and didn't find any similar bug report.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by joncampbell123** [comment]
None of the ViRGE 3D acceleration functions are implemented yet, only the 2D acceleration functions.

**Comment by BridgeHeadland** [comment]
@joncampbell123 Well, that explains why I couldn't start Return to Castle Wolfenstein in Windows XP last year. 😊
What are the minimum and recommended requirements for DOSBox-X, by the way? I couldn't start the correct build of DOSBox-X in Windows XP either.
